nntrainer/graph/network_graph.cpp

```diff
@@ -422,16 +424,43 @@ void NetworkGraph::backwarding(
     PROFILE_TIME_END(profile_keys.at(ln->getType()));
   }
 
+  auto update_loss_scale = [&](float scale) {
+    for (auto iter = iter_begin; iter != iter_end; iter++) {
+      auto &ln = *iter;
+      auto type = ln->getType();
+      if (type == CrossEntropyLossLayer::type ||
```

Comment:
I'm worried when the developer uses his loss layer. If he wants to use it.. then this loss scale seems not to work properly.

Reply:
It modified for loss layers to be grouped, and loss scale is controlled by the loss layers method.
Also, this type checking is removed for custom layer as you mentioned.

Reply:
Applying loss scale needs to be performed while derivative calculation, so it changed to just provide a method to apply loss scale.